From 169376c182c9ca786ce659b3e519f4017030c956 Mon Sep 17 00:00:00 2001
From: Jeffy Chen <jeffy.chen@rock-chips.com>
Date: Fri, 10 Dec 2021 10:24:29 +0800
Subject: [PATCH] aarch64: arm_acle.h: Fix compile error

There's a lonely induce in upstream GCC-9(since 9.4.0), which would cause
compile error:
error: template with C linkage

Signed-off-by: Jeffy Chen <jeffy.chen@rock-chips.com>
---
 gcc/config/aarch64/arm_acle.h | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/gcc/config/aarch64/arm_acle.h b/gcc/config/aarch64/arm_acle.h
index 5614735..8c622a3 100644
--- a/gcc/config/aarch64/arm_acle.h
+++ b/gcc/config/aarch64/arm_acle.h
@@ -98,10 +98,6 @@ __rint64x (double __a)
 
 #pragma GCC target ("+nothing+crc")
 
-#ifdef __cplusplus
-extern "C" {
-#endif
-
 __extension__ static __inline uint32_t __attribute__ ((__always_inline__))
 __crc32b (uint32_t __a, uint8_t __b)
 {
-- 
2.20.1

